primer-miso/Makefile

```diff
@@ -4,5 +4,11 @@
 
 include ../makefiles/common.mk
 
+# TODO The last line here shouldn't be necessary.
+# For some reason, GHCID is failing to discover the right paths to watch.
```

Comment:
I tried to look in to why we need to manually specify the paths. I think it's to do with using multiple components, but I still couldn't reproduce the issue in a simpler project. Anyway, I'd prefer to move to Ghciwatch eventually, but it doesn't yet support multiple components at all: https://github.com/MercuryTechnologies/ghciwatch/issues/316.

Reply:
Also, I considered adding `lib:primer` and `--reload ../primer/src`, in order to also pick up changes to the core library. But this brings the minimum reload time from about 0.5s on my machine to around 3s, even when the only changes are in the frontend components.

Without any of these issues, the command could be as simple as `ghcid -c "cabal repl all" -Wr --reload frontend/style.css`.